### PR TITLE
[INLONG-568][Doc] Create the configuration document for manager

### LINKED ISSUE #568: [Feature][Doc] Create the configuration document for manager
### Description

[Feature][Doc] Create the configuration document for manager

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request
Create the configuration document for manager
- Fixes #568 

### Motivation

Create the configuration document for manager
### Modifications

Create the configuration document for manager
### Verifying this change

- [ ] Make sure that the change passes the CI checks.

### Documentation

  - Does this pull request introduce a new feature? (no)

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+| spring.datasource.druid.connectionProperties       | 数据库连接属性          | druid.stat.mergeSql=true;druid.stat.slowSqlMillis=5000                                                                                                                  | 通过connectProperties属性来打开mergeSql功能；慢SQL记录                         |
+
+## 审计配置
+
+- 本节介绍在给定配置文件中指定的重要参数：
+- application-dev.properties, application-prod.properties, application-test.properties
+
+| 参数                               | 值         | 默认值   | 注释                                                 |
+|----------------------------------|-----------|-------|----------------------------------------------------|
+| audit.query.source   | MYSQL/ELASTICSEARCH/CLICKHOUSE | MYSQL | 决定要查询的数据源的审计信息，当前仅支持MYSQL,ELASTICSEARCH,CLICKHOUSE |
```
Comment (healchow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md:79]: 在中英文之间加一个逗号，另外标点符号使用中文。

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+- 本节介绍在给定配置文件中指定的重要参数：
+- application-dev.properties, application-prod.properties, application-test.properties
+
+| 参数                               | 值               | 默认值  | 注释                                                   |
+|----------------------------------|-----------------|------|------------------------------------------------------|
+| logging.level.{effective_area}   | info/warn/error | info | effective_area是日志级别的有效面积，root代表整个项目。它还可以设置为某个包的全限定类名 |
+
+## 数据库配置
+
+- 本节介绍在给定配置文件中指定的重要参数：
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md:51]: this line is needless?

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+| spring.datasource.druid.testOnReturn               | true/false       | false                                                                                                                                                                   | 释放连接到连接池时，是否检测连接可用性，开启会对性能造成一定影响                                  |
+| spring.datasource.druid.filters                    | stat,wall,log4j  | stat,wall                                                                                                                                                               | 监控统计用的filter:stat日志用的filter:log4j防御sql注入的filter:wall  'wall'用于防火墙 |
+| spring.datasource.druid.connectionProperties       | 数据库连接属性          | druid.stat.mergeSql=true;druid.stat.slowSqlMillis=5000                                                                                                                  | 通过connectProperties属性来打开mergeSql功能；慢SQL记录                         |
+
+## 审计配置
+
+- 本节介绍在给定配置文件中指定的重要参数：
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md:74]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+- application-dev.properties, application-prod.properties, application-test.properties
+
+| 参数                               | 值         | 默认值   | 注释                                                  |
+|----------------------------------|-----------|-------|-----------------------------------------------------|
+| audit.query.source   | MYSQL/ELASTICSEARCH/CLICKHOUSE | MYSQL | 决定要查询的数据源的审计信息，当前仅支持，MYSQL，ELASTICSEARCH，CLICKHOUSE |
+
+## Elasticsearch配置
+
+- 本节介绍在给定配置文件中指定的重要参数：
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md:83]: ditto

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+## 审计配置
+
+- application-dev.properties, application-prod.properties, application-test.properties
+
+| 参数                               | 值         | 默认值   | 注释                                                  |
+|----------------------------------|-----------|-------|-----------------------------------------------------|
+| audit.query.source   | MYSQL/ELASTICSEARCH/CLICKHOUSE | MYSQL | 决定要查询的数据源的审计信息，当前仅支持，MYSQL，ELASTICSEARCH，CLICKHOUSE |
+
+## Elasticsearch配置
```
Comment (dockerzhang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/modules/manager/configure.md:78]: Elasticsearch配置
->
Elasticsearch 配置

add more space after English words.

**Comment by dockerzhang** [comment]
please add `sidebar_position: ` to keep the order.

**Review by dockerzhang — APPROVED** [review]

File: docs/modules/manager/configure.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+| audit.query.source   | MYSQL/ELASTICSEARCH/CLICKHOUSE                | MYSQL   | Audit query source that decide what data source to query, currently only supports MYSQL, ELASTICSEARCH, CLICKHOUSE |
+| es.index.search.hostname  | Elasticsearch hostname                        | 127.0.0.1 | Elasticsearch host split by coma if more than one host, such as 'host1,host2'                                          |
+| es.index.search.port      | Elasticsearch port                            | 9200      |                                           |
+| es.auth.enable            | true/false                                    | false     | Elasticsearch support authentication flag |
+| es.auth.user              | Elasticsearch user of authentication info     | admin     |                                           |
+| es.auth.password          | Elasticsearch password of authentication info | inlong    |                                           |
+| audit.ck.jdbcUrl        | Clickhouse jdbc url                           | jdbc:clickhouse://127.0.0.1:8123/apache_inlong_audit |                                   |
```
Comment (healchow) [inline docs/modules/manager/configure.md:81]: `Clickhouse` -> `ClickHouse`.

**Review by healchow — APPROVED** [review]

**Pull request merged**